src/components/MapView/utils.ts

```diff
@@ -1,7 +1,10 @@
+import _ from 'lodash';
```

Comment:
we should import the exact functions we use with `{uniq, ...} from ...` since it's easier to "tree-shake" in the future (reduce bundle size)

Reply:
Yeah. I have solved it. Check below

```javascript
import { values, zipObject, assign, difference, mapValues, groupBy, keysIn, keyBy, uniq } from 'lodash';
```